src/debuginfo.cpp

```diff
@@ -1112,6 +1115,14 @@ static int jl_getDylibFunctionInfo(jl_frame_t **frames, size_t pointer, int skip
                                               pointer, &pip, NULL) == 0)
             saddr = (void*)pip.start_ip;
 #endif
+#if defined(_OS_WINDOWS_) && defined(_CPU_X86_64_)
```

Comment:
RtlLookupFunctionEntry was introduced for the x64 ABI. There is no equivalent fix available for x32.